bus_board.rb

```diff
@@ -0,0 +1,71 @@
+require 'json'
+require 'net/http'
+require 'uri'
+
+def display_bus(bus)
+  time_until_arrival = bus["timeToStation"]/60 == 0 ? "due" : "#{(bus["timeToStation"]/60).to_s} minutes"
```

Comment:
Should we define some constants instead of integers? e.g. SECONDS_IN_A_MINUTE, NUMBER_OF_TIMES_TO_DISPLAY?